nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -517,7 +541,8 @@ def parse_cmd_args(cmd, arg):
                                   name=name,
                                   command=command,
                                   volume_mounts=volume_mounts,
-                                  args=args)
+                                  image_pull_policy=image_pull_policy,
```

Comment:
If the method/function `checked_image_pull_policy()`is made generic as suggested, I would use it here.
```suggestion
                                  image_pull_policy=self.get_image_pull_policy(image_pull_policy),
```

Reply:
Hmm... I cannot use the static method inside another static method.
I use the "image_pull_policy=self.job_image_pull_policy," as the check is already performed for this variable.